### PR TITLE
New QBox and add Dutch locale

### PR DESCRIPTION
**Pull request by StrixLuca**
## Description

Add Dutch locale and Add new qbox 

## Checklist

- [nope ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ yes] My code fits the style guidelines.
- [ yes] My PR fits the contribution guidelines.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@ lib.locale()
 
 -- Variables
 
-local PlayerData = QBX.Functions.GetPlayerData()
+local PlayerData = exports.qbx_core:GetPlayerData()
```
Comment (Manason) [inline client/main.lua]:
```suggestion
```
Use QBX.PlayerData where needed

File: client/main.lua [hunk 2]
```diff
@@ -296,7 +296,7 @@ AddEventHandler('onResourceStart', function(resourceName)
 end)
 
 RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
-    PlayerData = QBX.Functions.GetPlayerData()
+    PlayerData = exports.qbx_core:GetPlayerData()
```
Comment (Manason) [inline client/main.lua]:
```suggestion
```
player data module will take care of updating PlayerData

**Review by Manason — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (StrixLuca) [reply client/main.lua]: ahw oke 
Thanks

**Comment by StrixLuca** [comment]
what i need to change with  



RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
    PlayerData = {}
    updateBlip()
    updateZone()
end)

RegisterNetEvent('QBCore:Player:SetPlayerData', function(val)
    PlayerData = val
    updateBlip()
    updateZone()
end)

**Comment by StrixLuca** [comment]
i think is good now

**Review by Manason — APPROVED** [review]

**Pull request merged**